Misc documentation updates (#132)
stencila/designa: packages/components/readme.md

## Using these components

### Add to your project

> There are two ways to load these components into your project, a `script` tag in an HTML file, or as an `npm` pacakge.

Reviewer: ```suggestion
There are two ways to load these components into your project, a `script` tag in an HTML file, or as a Node module.
```

#### Script tag

- Put the following script tags in the head of your HTML document:

```js
<script type="module" src="https://unpkg.com/@stencila/components@latest/dist/stencila-components/stencila-components.esm.js"></script>
<script type="text/javascript" nomodule="true" src="https://unpkg.com/@stencila/components@latest/dist/stencila-components/stencila-components.js"></script>
```

> - Then you can use the element anywhere in your template, JSX, html, etc.

Reviewer: ```suggestion
- Then you can use the element anywhere in your HTML, HTML template, JSX, etc.
```

> For modern browser support the `module` attribute, only the components used on the page will be requested and lazy-loaded, keeping the page size low.

Reviewer: ```suggestion
In modern browsers supporting the `module` attribute, only the components used on the page will be requested and lazy-loaded, keeping the page size low.
```

> #### Node Modules

Reviewer: ```suggestion
#### Node module
```

- Run `npm install @stencila/components --save`
- Import the package in your main/top-level file: `import '@stencila/components'`

> - Stencil will automatically setup the lazy-loading capabilities for the Stencil library. Then you can use the element anywhere in your template, JSX, html etc.

Reviewer: ```suggestion
- Stencil will automatically setup the lazy-loading capabilities for the Stencil library. Then you can use the element anywhere in your template, JSX, HTML etc.
```

### Using and customizing components

> The best reference for how to use and combine these components is our [component library](https://stencila.github.io/designa/?path=/story/schema-nodes-code-chunk--with-execute-handler).

Reviewer: ```suggestion
The best reference for how to use and combine these components is our [component library](https://stencila.github.io/designa/).
```
